content/v2.0/query-data/flux/calculate-percentages.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Calculate percentages with Flux
+list_title: Calculate percentages
+description: >
+  Use [`pivot()` or `join()`](/v2.0/query-data/flux/calculate-percentages/#pivot-vs-join)
+  and the [`map()` function](/v2.0/reference/flux/stdlib/built-in/transformations/map/)
+  to align operand values into row-wise sets and calculate a percentage.
+menu:
+  v2_0:
+    name: Calculate percentages
+    parent: Query with Flux
+weight: 220
+aliases:
+ - /v2.0/query-data/guides/manipulate-timestamps/
+related:
+  - /v2.0/query-data/flux/mathematic-operations
+  - /v2.0/reference/flux/stdlib/built-in/transformations/map
+  - /v2.0/reference/flux/stdlib/built-in/transformations/pivot
+  - /v2.0/reference/flux/stdlib/built-in/transformations/join
+list_query_example: percentages
+---
+
+
+Calculating percentages using multiple values in a queried data set is a common use case for time series data.
+With Flux, all operand values need to exists in a single row to use them in a mathematic calculation.
+Once operands are aligned in rows, use `map()` to re-map values in the row and calculate a percentage.
+
+**To calculate percentages:**
+
+1. Query the necessary operand values.
+2. Use [`pivot()` or `join()`](#pivot-vs-join) to align operand values into row-wise sets.
+3. Use [`map()`](/v2.0/reference/flux/stdlib/built-in/transformations/map/)
+   to divide the numerator operand value by the denominator operand value and multiply by 100.
+
+{{< code-tabs-wrapper >}}
+{{% code-tabs %}}
+[pivot()](#)
+[join()](#)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+```js
+from(bucket: "example-bucket")
+  |> range(start: -1h)
+  |> filter(fn: (r) => r._measurement == "m1" and r._field =~ /field[1-2]/ )
+  |> pivot(rowKey:["_time"], columnKey: ["_field"], valueColumn: "_value")
+  |> map(fn: (r) => ({ r with _value: r.field1 / r.field2 * 100.0 }))
+```
+{{% /code-tab-content %}}
+{{% code-tab-content %}}
+```js
+t1 = from(bucket: "example-bucket")
+  |> range(start: -1h)
+  |> filter(fn: (r) => r._measurement == "m1" and r._field == "field1")
+
+t2 = from(bucket: "example-bucket")
+  |> range(start: -1h)
+  |> filter(fn: (r) => r._measurement == "m2" and r._field == "field2")
+
+join(tables: {t1: t1, t2: t2}, on: ["_time"])
+  |> map(fn: (r) => ({ r with _value: r._value_t1 / r._value_t2 * 100.0 }))
+```
+{{% /code-tab-content %}}
+{{< /code-tabs-wrapper >}}
+
+{{% note %}}
+This guide uses `pivot()` to align operand values into row-wise sets because
+`pivot()` works for the majority of use cases and is more performant than `join()`.
+_See [Pivot vs join](#pivot-vs-join) below._
+{{% /note %}}
+
+## GPU monitoring example
+The following example queries data collected from a GPU monitoring solution and
+calculates the percentage of GPU memory used over time.
+Data includes the following:
+
+- **`gpu` measurement**
+- **`mem_used` field**: used GPU memory in bytes
+- **`mem_total` field**: total GPU memory in bytes
```

Comment:
Reword as numbered steps to indicate required order?

Reply:
These aren't ordered steps. These are just points describing the schema.

Reply:
Ah, sorry, my comment wasn't too clear...my thought was it may be helpful to indicate order (think you did that above in first procedure), like:
1. Specify `gpu-monitor` bucket.
...
3. Use `filter()` to specify the `gpu` measurement and two fields to query: `mem_used` and `mem_total `